/*---------------------------------------------------
Rodrigo Mungu√≠a 2021.

System parameters
-----------------------------------------------------
*/
#include <string>
#include <armadillo>

#ifndef PARAMETERS_GL_H
#define PARAMETERS_GL_H

using namespace std;



struct parameters
{
    
        


} ;   





#endif /*PARAMETERS_H*/